### PR TITLE
sync-diff: update passwd description

### PR DESCRIPTION
**Pull request by lichunzhu**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Update description about `sync-diff-inspector`'s password configuration for base64 encoded password.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tidb-tools/pull/688

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by TomShawn — APPROVED** [review]
LGTM

**Review by niubell — APPROVED** [review]
LGTM

File: sync-diff-inspector/sync-diff-inspector-overview.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ check-struct-only = false
     host = "127.0.0.1"
     port = 4000
     user = "root"
-    password = ""
+    password = "" # 设置连接数据库的密码，可为明文或 Base64 编码。
```
Comment (Oreoxmt) [inline sync-diff-inspector/sync-diff-inspector-overview.md:102]:
```suggestion
    password = "" # 设置连接下游数据库的密码，可为明文或 Base64 编码。
```

File: sync-diff-inspector/sync-diff-inspector-overview.md [hunk 2]
```diff
@@ -90,7 +90,7 @@ check-struct-only = false
     host = "127.0.0.1"
     port = 3306
     user = "root"
-    password = ""
+    password = "" # 设置连接数据库的密码，可为明文或 Base64 编码。
```
Comment (Oreoxmt) [inline sync-diff-inspector/sync-diff-inspector-overview.md:93]:
```suggestion
    password = "" # 设置连接上游数据库的密码，可为明文或 Base64 编码。
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Pull request merged**